pages/stats.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react'
+import Page from '../components/page'
+import Title from '../components/title'
+import GoBack from '../components/goBack'
+import PlayerDetails from '../components/playerDetails'
+import SkaterStats from '../components/SkaterStats'
+import GoalieStats from '../components/goalieStats'
+// import GoalieStats from '../components/GoalieStats'
+import { retrievePlayersStats } from '../utils/stats'
+
+export default ({ location }) => {
+  const [playersLastName, setPlayersLastName] = useState('')
+  const [player, setPlayer] = useState({})
+  const [skaterStats, setSkaterStats] = useState({})
+  const [goalieStats, setGoalieStats] = useState({})
+
+  useEffect(() => {
+    async function pullData() {
+      const { skaterStat, goalieStat, details } = await retrievePlayersStats(location)
+
+      setPlayersLastName(details.lastName)
+      setPlayer(details)
+      setSkaterStats(skaterStat)
+      setGoalieStats(goalieStat)
+    }
+
+    pullData()
+  }, [])
+  if (!playersLastName) {
+    return (<div>Sorry, player not found</div>)
+  }
```

Comment:
Doing it this way means your error page is missing all your styling and a go back link. Should be part of the return